components/financial_assistance/spec/domain/financial_assistance/operations/applicant/create_or_update_spec.rb

```diff
@@ -158,7 +158,7 @@
       end
 
       let!(:applicant_params) do
-        default_tobacco_use = EnrollRegistry.feature_enabled?(:sensor_tobacco_carrier_usage) ? "U" : "unknown"
```

Comment:
I know we want to update the string based on the call earlier, but should this not be flagged?